### PR TITLE
Fix CI unit tests

### PR DESCRIPTION
**Pull request by samerd**
Signed-off-by: Samer Deeb <samerd@mellanox.com>

##### SUMMARY

Fix unit tests: add missing functions

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: .github/workflows/ansible-test.yml [hunk 1]
```diff
@@ -17,6 +17,9 @@ jobs:
         with:
           python-version: 3.6
 
+      - name: Install ansible.netcommon
```
Comment (gundalow) [inline .github/workflows/ansible-test.yml:20]: Needs to be after you install ansible

Comment (gundalow) [reply .github/workflows/ansible-test.yml:20]: From the logs, we can see this is currently being installed to `Installing 'ansible.netcommon:1.0.0' to '/home/runner/.ansible/collections/ansible_collections/ansible/netcommon`

Add ` -p ../../`

File: .github/workflows/ansible-test.yml [hunk 2]
```diff
@@ -39,6 +42,9 @@ jobs:
       - name: Install ansible-base (devel)
         run: pip install https://github.com/ansible/ansible/archive/devel.tar.gz --disable-pip-version-check
 
+      - name: Install ansible.netcommon
+        run: ansible-galaxy collection install ansible.netcommon
```
Comment (gundalow) [inline .github/workflows/ansible-test.yml:46]: Add ` -p ../../`

**Comment by gundalow** [comment]
@samerd FYI I've pushed a commit which should fix `ansible.netcommon` so you will need to `git pull` locally

**Comment by gundalow** [comment]
Remove these lines
```
ERROR: tests/sanity/ignore-2.11.txt:8:1: A100: Ignoring 'nonexistent-parameter-documented' on 'plugins/modules/onyx_command.py' is unnecessary
ERROR: tests/sanity/ignore-2.11.txt:12:1: A100: Ignoring 'nonexistent-parameter-documented' on 'plugins/modules/onyx_config.py' is unnecessary

ERROR: Found 6 ignores issue(s) which need to be resolved:
ERROR: tests/sanity/ignore-2.11.txt:70:1: File 'plugins/modules/onyx_ntp.py' does not exist
ERROR: tests/sanity/ignore-2.11.txt:71:1: File 'plugins/modules/onyx_ntp_servers_peers.py' does not exist
ERROR: tests/sanity/ignore-2.11.txt:90:1: File 'plugins/modules/onyx_snmp.py' does not exist
ERROR: tests/sanity/ignore-2.11.txt:91:1: File 'plugins/modules/onyx_snmp_hosts.py' does not exist

ERROR: tests/sanity/ignore-2.11.txt:92:1: File 'plugins/modules/onyx_snmp_users.py' does not exist
ERROR: tests/sanity/ignore-2.11.txt:93:1: File 'plugins/modules/onyx_syslog_remote.py' does not exist
```

**Comment by samerd** [comment]
thanks so much for the review and the help @gundalow 
much appreciated

**Comment by gundalow** [comment]
@samerd Great work!

**Pull request merged**